docs/rules/require-meta-fixable.md

```diff
@@ -44,20 +44,6 @@ module.exports = {
 };
 ```
 
-```js
-/* eslint eslint-plugin/require-meta-fixable: "error" */
-
-module.exports = { create (context) {
-  context.report({
-    node,
-    message: 'foo',
-    fix (fixer) {
-      return fixer.remove(node);
-    },
-  });
-} };
-```
-
```

Comment:
why removing the example? it seems a good example to have.

Reply:
This example shows the rule without any `meta` object. I would this to be a rare edge case, and I would argue that including too many unimportant edge cases can distract from the primary examples and make the documentation too long and difficult to understand. The key point we're trying to get across is that `fixable` being missing is in invalid case, and that's already covered by the test case above this one.

If you still prefer, I can add it back, but just wanted to make my case against it.